examples/step-31/step-31.cc

```diff
@@ -401,9 +402,19 @@ namespace Step31
       :
       stokes_matrix           (&S),
       m_inverse               (&Mpinv),
-      a_preconditioner        (Apreconditioner),
-      tmp                     (stokes_matrix->block(1,1).m())
-    {}
+      a_preconditioner        (Apreconditioner)
+    {
+      // When using a TrilinosWrappers::MPI::Vector or a
+      // TrilinosWrappers::MPI::BlockVector, the Vector is initialized using an
+      // IndexSet. IndexSet is used not only to resize the
+      // TrilinosWrappers::MPI::Vector but it also associates an index in the
+      // TrilinosWrappers::MPI::Vector with a degree of freedom (see step-40 for
+      // a more detailed explanation). This assocation is done by the add_range()
+      // function.
```

Comment:
can you say here that this produces the "complete index set where every valid index is part of the set", or similar?